docs/docs/migration.mdx

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Nit: there are quite a few instances of `1.0` and `2.0` in this doc. I would recommend replacing them with `1.x` and `2.x` everywhere.